### PR TITLE
displaying all chatrooms in a table format

### LINKED ISSUE #7: As a user, i want to see the list of all the chatrooms in a table format.

### PR DESCRIPTION
**Pull request by Harmeet011**
Hi, I have added a table of all the chatrooms with id, name, members and created By columns.

### CODE REVIEW

File: src/components/ChatRoom.js [hunk 1]
```diff
@@ -1,38 +1,56 @@
...
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  return (
+    <div>
+      <h2>Chat Rooms</h2>
+      <Table striped bordered hover>
```
Comment (RichaChauhan1989) [inline src/components/ChatRoom.js:61]: Hi, can you please show a loading spinner while the chatrooms are being loaded.

File: src/components/ChatRoom.js [hunk 2]
```diff
@@ -1,38 +1,56 @@
...
+            <th>Created By</th>
+          </tr>
+        </thead>
+        <tbody>
+          {chatrooms.map((chatroom) => (
+            <tr key={chatroom.id}>
+              <td>{chatroom.id}</td>
+              <td>{chatroom.name}</td>
+              <td>{chatroom.members}</td>
+              <td>{chatroom.created_by}</td>
```
Comment (RichaChauhan1989) [inline src/components/ChatRoom.js]: We need to show the members name in place of Id

**Comment by RichaChauhan1989** [comment]
- [x] Reviewed Code Changes
- [x] Constructive Feedback
- [x]  Issues resolved
- [x] Pull Request Completed
- [x] Integration Testing(Selenium)
- [x] Build complete
- [x] Deployed to Production

**Comment by param1910** [comment]
Anyone got the email of presentation?

**Review by param1910 — DISMISSED** [review]

File: src/components/ChatRoom.js [hunk 3]
```diff
@@ -1,38 +1,56 @@
...
+
+  return (
+    <div>
+      <h2>Chat Rooms</h2>
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Id</th>
+            <th>Chatroom Name</th>
+            <th>Members</th>
```
Comment (param1910) [inline src/components/ChatRoom.js]: Can you please rename this column to Chatroom Members as we discussed.

File: src/components/ChatRoom.js (hunk 2 above)
Comment (param1910) [inline src/components/ChatRoom.js]: We also need to display Created by username instead of Id, currently it shows Id of user.

**Review by param1910 — APPROVED** [review]

**Review by RichaChauhan1989 — APPROVED** [review]
Great Job!

**Pull request merged**